File: componet/src/main/java/org/zgl/service/client/db/BankOperation.java
package org.zgl.service.client.db;

import org.zgl.dto.clinet.hall.BankDto;
import org.zgl.proxy.rule.IHttpSyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/21
 * @文件描述：
 */
public interface BankOperation extends IHttpSyncService {
    BankDto save(long uid, long gold);
    BankDto draw(long uid, long gold);
}

File: cs/proxy/db/BankOperation.cs
/// <summary>
/// 
/// </summary>
namespace org.zgl.service.client.db{
	[Proxy(gameId = -7)]
	public interface BankOperation : IHttpSyncService { 
		/// <summary>
		/// 
		/// </summary>
		BankDto save(long arg0,long arg1);
		/// <summary>
		/// 
		/// </summary>
		BankDto draw(long arg0,long arg1);
	}
}
